src/client/Microsoft.Identity.Client/OAuth2/TokenClient.cs

```diff
@@ -56,6 +56,12 @@ public async Task<MsalTokenResponse> SendTokenRequestAsync(
                 {
                     tokenEndpoint = await _requestParams.Authority.GetTokenEndpointAsync(_requestParams.RequestContext).ConfigureAwait(false);
                 }
+                if (!string.IsNullOrEmpty(_requestParams.FmiPathSuffix))
+                {
+                    //Need to verify additional "/" is not added
+                    tokenEndpoint += _requestParams.FmiPathSuffix;
```

Comment:
Are there apis that can already append things to the end of the token endpoint? I dont recall seeing anything like that.

Reply:
No no, you just need to add it as a body parameter. Similar to client_assertion and client_id. The token endpoint does not change.